src/Bicep.Core.UnitTests/Utils/OutputHelper.cs

```diff
@@ -71,7 +71,12 @@ public static string GetDiagLoggingString(string sourceText, string outputDirect
             {
                 message = Regex.Replace(message, @"'\${TEST_OUTPUT_DIR}.*?'", new MatchEvaluator((match) => match.Value.Replace('\\', '/')));
             }
-            return $"[{diagnostic.Code} ({diagnostic.Level})] {message} |{spanText}|";
+
+            var docLink = diagnostic.Uri == null
+                ? string.Empty
+                : $"{diagnostic.Source}({diagnostic.Uri})";
```

Comment:
Are Source & Uri related to each other? Wouldn't it make sense to log them independently?

Reply:
This is the way LSP creates the message. Just replicating that look.